packages/docs/pages/reference/elements/fillin.mdx

```diff
@@ -8,12 +8,48 @@ import {
 
 # `<fillin>`
 
-Short description of what the `<fillin>` tag does.
+The self-closing `<fillin/>` tag is a generic fill-in-the-blank tag 
+which can produce answer blanks as underline, boxes or shade.
 
 ## Syntax
 
 <SyntaxDisplay name="fillin" />
 
+### Remark
+`<fillin>` has publication options which govern the style 
+of fillin in both text and math modes.  
+ Defaults are shown in the examples.
+
 ## Examples
 
-    
\ No newline at end of file
+### A `<fillin/>` in math
+
+```ptx-example
+<FRAGMENT parents="p">
+    <c>&lt;m&gt;x^a * x^b = x^&lt;fillin fill="a+b" /&gt;&lt;/m&gt;</c> produces
```

Comment:
I think it's more confusing than helpful to have the `<c>` in here. The extra `&lt;` entities make it harder to understand the code. Plus, they see the code and its rendered form already, so there is no need for "meta" examples.